cpp/server/event.cc

```diff
@@ -183,7 +183,7 @@ bool EventLoop::EraseCheck(std::deque<FD*>::iterator* pfd) {
 // static
 void EventLoop::Set(int fd, fd_set* fdset, int* max) {
   DCHECK_GE(fd, 0);
-  CHECK_LT((unsigned)fd, FD_SETSIZE);
+  CHECK_LT((unsigned)fd, (unsigned)FD_SETSIZE);
```

Comment:
I don't think FD_SETSIZE is a fixed type and this file uses non c++ casts elsewhere.